packages/babel-traverse/src/scope/binding.ts

```diff
@@ -113,18 +146,19 @@ export default class Binding {
     if (this.referencePaths.indexOf(path) !== -1) {
       return;
     }
-    this.referenced = true;
-    this.references++;
     this.referencePaths.push(path);
   }
 
   /**
    * Decrement the amount of references to this binding.
    */
 
-  dereference() {
-    this.references--;
-    this.referenced = !!this.references;
+  dereference(path: NodePath) {
```

Comment:
If `path` is not defined, can we default it to the last element of `referencePaths`? So the previous usage `NodePath#derefence()` is still respected.